deployments/helm/hephaestus/values.yaml

```diff
@@ -24,6 +24,15 @@ enablePodSecurityPolicies: false
 # https://cert-manager.io/docs/faq/cluster-resource/
 certManagerNamespace: cert-manager
 
+# Annotations applied to pods
+podAnnotations: {}
+
+# Labels applied to pods
+podLabels: {}
+
+# Environment applied to pods
+podEnv: {}
+
```

Comment:
Maybe just indicate `applied to all pods`? It's implicit since it's a top-level config but I always try to error on the side of being explicit.